integrations/mistral/tests/test_mistral_chat_generator.py

```diff
@@ -0,0 +1,314 @@
+import os
+from datetime import datetime
+from typing import Iterator
+from unittest.mock import patch
+
+import pytest
+import pytz
+from haystack.components.generators.utils import print_streaming_chunk
+from haystack.dataclasses import ChatMessage, StreamingChunk
+from haystack.utils.auth import Secret
+from haystack_integrations.components.generators.mistral.chat.chat_generator import MistralChatGenerator
+from openai import OpenAIError, Stream
```

Comment:
`openai` is not a dependency on `pyproject.toml`

Reply:
wait, I'm wrong, it's another issue - I think it should be added here

```
[[tool.mypy.overrides]]
module = [
  "mistral.*",
  "haystack.*",
  "haystack_integrations.*",
  "pytest.*",
  "numpy.*",
]
```

I get this error when running the tests locally:

```
tests/test_mistral_chat_generator.py:12: error: Cannot find implementation or library stub for module named "openai"  [import-not-found]
tests/test_mistral_chat_generator.py:13: error: Cannot find implementation or library stub for module named "openai.types.chat"  [import-not-found]
tests/test_mistral_chat_generator.py:14: error: Cannot find implementation or library stub for module named "openai.types.chat.chat_completion_chunk"  [import-not-found]

```

Reply:
Hey @davidsbatista - Just so that I'm understsanding correctly, should I do the following? This one confused me because isn't openai a dependency of haystack-ai anyway? Why would we need to install it separately. Educate me pls 🙏 
```
[[tool.mypy.overrides]]
module = [
  "mistral.*",
  "haystack.*",
  "haystack_integrations.*",
  "openai.*",
  "pytest.*",
  "numpy.*",
]
```

Reply:
It's not related to the dependencies; my bad, I thought it was.

This is `mypy`, saying that it could not find type annotations for that lib; it often happens with 3rd party libs, so we are telling `mypy` to ignore it.